src/chapters/inheritance/exercises.rst

```diff
@@ -0,0 +1,11 @@
+Exercises: Inheritance
+======================
+
+Work on these exercises in the IntelliJ ``java-web-dev-exercises`` project.
+
+1. **Class design:** You are given a ``Computer`` class and need to add two additional classes: ``Laptop`` and ``SmartPhone``.
```

Comment:
i know you mentioned that the student have done a lot of class designing at this point, but maybe this is an area where we can give them light requirements, like give the computer at least 3 properties, 2 methods, one or more constructors, etc.

Reply:
Got it! Thanks!